### DISCUSSION TITLE
Increase limit for number of photos displayed at once

### DISCUSSION THREAD

**Discussion by timmeh-git** [jeremyknight-me/photocollage#16] [Ideas]
The current limit of 30 photos doesn't fill a multi-monitor setup well when using the default photo size. Can this limit be increased?

**Comment by jeremyknight-me** [comment]
That limit can definitely be increased. If you want to do it now, go to the following location and edit the NumberOfPhotos entry.

`C:\Users\YourUserName\AppData\Local\DigitalLagniappe\Screensavers\PhotoCollage\photo-collage.config`

If you give it more than 50, please let me know how it runs. I have never tested past 30 and would love the feedback. I finally have a third monitor, so I need to re-run all kinds of tests anyway so when I have time, I'll do some testing at 50 and then boost it to 50 in the options window if all the tests pass.

**Comment by timmeh-git** [comment]
That's great thanks. I've tried it out. Performance wise I tried 50 and 70 and both were fine however once the cap was reached some new images would just flash and disappear leaving behind what looks like a single pixel with the border around it. This happened rarely at 50 but happened a lot more at 70.

In terms of screen fill I upped it to 70 because 50 wasn't filling all screens with the default (500) image size. This is partly to do with the fact that approximately 20% of the images are loaded completely off screen. Also some areas of my larger screen seemed to be rarely populated so I always had a few black areas. Not sure if there's any way of adjusting the algorithm to actually do anything about that. It may be less of an issue when images can be forced to load on screen rather than off screen.

Anyway I re-tested using 800 size for the images and it filled a bit better for now.

For reference I am running one ultrawide screen at 3440x1440 and two standard wide screens at 2560x1440 above the ultrawide.

**Comment by jeremyknight-me** [comment]
merged in #19 which will give the option to go to 100 :)
